spec/controllers/measures_controller_spec.rb

```diff
@@ -10,29 +10,30 @@
     let!(:draft_measure) { FactoryBot.create(:measure, draft: true) }
 
     context "when not signed in" do
-      it { expect(subject).to be_ok }
-
-      it "all published measures (no drafts)" do
-        json = JSON.parse(subject.body)
-        expect(json["data"].length).to eq(1)
-      end
+      it { expect(subject).to be_forbidden }
     end
 
     context "when signed in" do
       let(:guest) { FactoryBot.create(:user) }
       let(:user) { FactoryBot.create(:user, :manager) }
+      let(:analyst) { FactoryBot.create(:user, :analyst) }
 
-      it "guest will not see draft measures" do
+      it "guest will be forbidden" do
         sign_in guest
-        json = JSON.parse(subject.body)
-        expect(json["data"].length).to eq(1)
+        expect(subject).to be_forbidden
       end
 
       it "manager will see draft measures" do
         sign_in user
         json = JSON.parse(subject.body)
         expect(json["data"].length).to eq(2)
       end
+
+      it "analyst will see draft measures" do
```

Comment:
please note: analysts should generally NOT be able to see "draft" content (draft = false)